src/nbl/asset/interchange/CHLSLLoader.cpp

```diff
@@ -38,6 +38,12 @@ SAssetBundle CHLSLLoader::loadAsset(system::IFile* _file, const IAssetLoader::SA
 		{".comp.hlsl",IShader::E_SHADER_STAGE::ESS_COMPUTE},
 		{".mesh.hlsl",IShader::E_SHADER_STAGE::ESS_MESH},
 		{".task.hlsl",IShader::E_SHADER_STAGE::ESS_TASK},
+		{".rgen.hlsl",IShader::E_SHADER_STAGE::ESS_RAYGEN},
+		{".rahit.hlsl",IShader::E_SHADER_STAGE::ESS_ANY_HIT},
+		{".rchit.hlsl",IShader::E_SHADER_STAGE::ESS_CLOSEST_HIT},
+		{".rmiss.hlsl",IShader::E_SHADER_STAGE::ESS_MISS},
+		{".rint.hlsl",IShader::E_SHADER_STAGE::ESS_INTERSECTION},
+		{".rcall.hlsl",IShader::E_SHADER_STAGE::ESS_CALLABLE},
 	};
 	auto shaderStage = IShader::E_SHADER_STAGE::ESS_UNKNOWN;
```

Comment:
ah one thing that #754 was missing was something like this, to let `.hlsl` only (no  `stage.hlsl`) extension map to library all-or-library stage